<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOWidget</class>
 <widget class="QWidget" name="IOWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>118</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="1">
       <widget class="HLX" name="widget_2" native="true"/>
      </item>
      <item row="0" column="6">
       <widget class="HLX" name="widget_5" native="true"/>
      </item>
      <item row="0" column="7">
       <widget class="HLX" name="widget_6" native="true"/>
      </item>
      <item row="0" column="8">
       <widget class="HLX" name="widget_7" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="HLX" name="widget_3" native="true"/>
      </item>
      <item row="0" column="9">
       <widget class="HLX" name="widget_8" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="HLX" name="widget_4" native="true"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="HLX" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HLX</class>
   <extends>QWidget</extends>
   <header location="global">hlx.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
